addons/mass_mailing/models/mailing.py

```diff
@@ -741,6 +755,11 @@ def _action_view_documents_filtered(self, view_filter):
             helper_header = _("No %s address bounced yet!", model_name)
             helper_message = _("Bounce happens when a mailing cannot be delivered (fake address, "
                                "server issues, ...). Check each record to see what went wrong.")
+        elif view_filter == 'clicked':
+            found_traces = self.mailing_trace_ids.filtered(lambda trace: trace.links_click_ids)
```

Comment:
A bit strange to me that we're using the relationships / filtered to find those traces when we could do a clean request on mailing traces with the appropriate domain.
Especially since there can be a LOT of them, right?

But I guess the other filters are also using the same logic, so maybe something to rework separately.

What do you think?